pages/index.vue

```diff
@@ -7,11 +7,11 @@
     <p class="text-lg pt-4 lg:pt-8 leading-loose font-serif text-gray-800">
       AlbionStatus is your <span class="text-black">reliable</span> Albion Online server status tracker! Operating
       since
-      <span class="text-black font-semibold">{{ $options.creationDate | moment('MMM Do YYYY') }}</span> (already
+      <span class="text-black font-semibold">{{ $options.creationDate | luxon }}</span> (already
       <span
         :title="$options.creationDate"
         class="text-black italic"
-      >{{ $options.creationDate | moment('from', new Date(), true) }}</span>
+      >{{ $options.creationDate | duration }}</span>
```

Comment:
This aligns well with the style before! However, I’d prefer a computed prop here though as the future of filters is uncertain. They were target of possible removal in Vue 2.x and might be in 3.x